### ISSUE TITLE
Marked Usage

### ISSUE THREAD

**Issue by janja-janja** [icio/sublime-text-marked#6]
Could you clarify on the usage on the md file please

**Comment by icio** [comment]
Could you please clarify the question? [The README has usage information](https://github.com/icio/sublime-text-marked#usage) -- is there a particular aspect of the plugin it's overlooking, or is its description unclear?

**Comment by janja-janja** [comment]
The description on the Marked usage is unclear. Could you explain a sample use of the package?

**Comment by icio** [comment]
What are you having problems with? Are you using OS X and do you have the Marked.app application installed? Open a markdown file in sublime and then either using the Tools → Marked menu, or "Marked" from the command palette (as per the usage instructions). It should open the current file in Marked.

**Comment by janja-janja** [comment]
Yes i am on OS X 10.10, i have Marked.app installed but i don't seem to get the working of it.
![screen shot 2015-05-07 at 09 38 16](https://cloud.githubusercontent.com/assets/9822841/7509687/c9d1e03e-f49c-11e4-8747-f827d59e5e6a.png)

**Comment by icio** [comment]
In your terminal, can you please try the following from your project directory?

``` bash
open -a Marked README.md
```

**Comment by Catta7140** [comment]
Hello

Have the same issue (no coder really) but command line works with:
open -a Marked\ 2 WORKdir/test.md

May that clarifies what's missing to make it work out from Sublime text 3 build 3083 / Marked 2.4.11 (895)

Thank you & Have a great day!

UPDATE:
Now it worked right away with second file -> new File -> Syntax MarkdownEditing -> Markdown GFM -> saved file as md - used Tools -> Marked voila (Sublime Text wasn't closed between first & second try, except opened first file via CLI as described above). Not sure what was the difference but put it between Keyboard and User. :-)
